miqa/core/tasks.py

```diff
@@ -25,7 +26,14 @@
 from miqa.learning.nn_classifier import evaluate1
 
 
-def evaluate_data(images: List[Image], session: Session):
+@app.task
+def evaluate_data(image_ids, session_id):
+    try:
+        images = [Image.objects.get(id=image_id) for image_id in image_ids]
```

Comment:
We can retrieve these with a single query rather than n queries using `filter(pk__in=image_ids)`